ledger/block/src/ratify/mod.rs

```diff
@@ -24,11 +24,14 @@ use indexmap::IndexMap;
 type Variant = u8;
 /// A helper type to represent the public balances.
 type PublicBalances<N> = IndexMap<Address<N>, u64>;
+/// A helper type to represent the bonded balances.
+type BondedBalances<N> = IndexMap<Address<N>, u64>;
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
I wonder if this has a non-trivial impact on object/storage size?

Reply:
It might have some impact, but the jump from `(Committee<N>, PublicBalances<N>)` to `(Committee<N>, PublicBalances<N>, BondedBalances<N>)` won't be that big.

Reply:
Considering every block contains this enum, and we will always pay the worst-case memory footprint, I don't understand why one chose to ignore this clippy warning.

You should follow the advise of `clippy` to resolve it (I assume its to use `Box`). Otherwise, every enum variant will always pay the worst case bound.

Reply:
> why one chose to ignore this clippy warning
By being too much in a get shit done mood, my bad.

@d0cd since you've been finishing off the PR, want to wrap the contents in Box? I also just recently noticed we're already using this pattern in snarkVM.

Reply:
Done! By boxing the elements of `Genesis::Ratify` the size reduces from 240 to 32 bytes.